src/test/java/org/openapitools/client/JsonParsingTest.java

```diff
@@ -0,0 +1,116 @@
+package org.openapitools.client;
```

Comment:
I know this issue basically forces it, but good idea for adding a testing section focused on some of the generated bits. 👍